### ISSUE TITLE
Support `:formatter` option for `Ougai::Logger.new`?

### ISSUE THREAD

**Issue by ybiquitous** [tilfin/ougai#88]
Hi, I'm wondering why `Ougai::Logger.new` method does not support the `:formatter` option, which the parent [`::Logger.new`](http://ruby-doc.org/stdlib-2.6.3/libdoc/logger/rdoc/Logger.html#method-c-new) can accept.

For example:

```ruby
logger = Ougai::Logger.new(STDOUT, formatter: Ougai::Formatters::Readable.new)
logger.formatter.class
#=> Ougai::Formatters::Bunyan

logger = ::Logger.new(STDOUT, formatter: Ougai::Formatters::Readable.new)
logger.formatter.class
#=> Ougai::Formatters::Readable
```

Ougai version is **1.8.0**.

It would be great if you could give me any response.
Thank you.

**Comment by tilfin** [comment]
The reason is that the original Logger constructor did not support the `:formatter` argument when Ougai was first released. I will fix that.

**Comment by ybiquitous** [comment]
Thank you so much!

**Comment by ybiquitous** [comment]
Thank you for your very quick fix! I confirmed the issue has been fixed by v1.8.1. 😊
